chain/arweave.ts

```diff
@@ -0,0 +1,76 @@
+import '../common/eager_offset'
+import { Bytes } from '..'
```

Comment:
You just have to change this import line to make the `yarn build` work.

```typescript
import { Bytes } from '../common/collections'
```

Also you should rebase with the current `main` branch 😉